packages/core/src/extensions/extension-loader/file-system-provisioner-store/ensure-hashed-directory-for-extension.injectable.ts

```diff
@@ -24,8 +25,15 @@ const ensureHashedDirectoryForExtensionInjectable = getInjectable({
     const directoryForExtensionData = di.inject(directoryForExtensionDataInjectable);
     const ensureDirectory = di.inject(ensureDirInjectable);
     const getHash = di.inject(getHashInjectable);
+    const findKeyContains = di.inject(findKeyContainsInjectable)<string>;
 
     return async (extensionName, registeredExtensions) => {
```

Comment:
Shouldn't this `extensionName` be the value from the `package.json`?

Reply:
yeah this has been changed to reconstruct the legacy key